gnosis/verification.k

```diff
@@ -6,21 +6,35 @@ module VERIFICATION
     imports EDSL
     imports LEMMAS
 
+  // ########################
+  // Memory Usage
+  // ########################
+
+    rule #memoryUsageUpdate(#memoryUsageUpdate(MU, START0, WIDTH0), START1, WIDTH1) => #memoryUsageUpdate(MU, START0, WIDTH0)
+      requires START1 +Int WIDTH1 <Int START0 +Int WIDTH0
+
+    rule #memoryUsageUpdate(#memoryUsageUpdate(MU, START0, WIDTH0), START1, WIDTH1) => #memoryUsageUpdate(MU, START1, WIDTH1)
+      requires START0 +Int WIDTH0 <=Int START1 +Int WIDTH1
+
+    rule 0 <=Int #memoryUsageUpdate(MU, START, WIDTH) => true
+
+    rule pow256 <=Int #memoryUsageUpdate(MU, START, WIDTH) => false  requires START +Int WIDTH <Int 3705346855594118253554271520278013051304639509300498049262642688253220148477921
+    rule #memoryUsageUpdate(MU, START, WIDTH) <Int  pow256 => true   requires START +Int WIDTH <Int 3705346855594118253554271520278013051304639509300498049262642688253220148477921
```

Comment:
could you put the comment explaining what this big number means?

Reply:
No longer needed. see https://github.com/kframework/evm-semantics/commit/da13d241f6521766a240227b00d1d02b7bf5a6ca